src/ert/callbacks.py

```diff
@@ -10,7 +10,7 @@
 from ert.run_arg import RunArg
 
 from .load_status import LoadResult, LoadStatus
-from .realization_state import RealizationState
+from .realization_state import RealizationState as RealizationStorageState
```

Comment:
I thought @xjules had renamed this already? Or I don't understand where `.` is here...

Reply:
Yes, on main! Still need to todo a major rebase :)